### DISCUSSION TITLE
2.07升级到2.09后服务起不来

### DISCUSSION THREAD

**Discussion by iso-lib** [AlistGo/alist#514] [Q&A]
我是用官方脚本直接安装的,2.07用的很好
curl -fsSL "https://nn.ci/alist.sh" | bash -s install

今天官方脚本升级
curl -fsSL "https://nn.ci/alist.sh" | bash -s update

停止 Alist 进程

下载 Alist  ...
######################################################################## 100.0%##O=#  #                                                                      

启动 Alist 进程

Alist 已更新到最新稳定版！

root@debian:~# systemctl status alist
● alist.service - Alist service
   Loaded: loaded (/lib/systemd/system/alist.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2022-02-08 08:10:52 EST; 3s ago
  Process: 25504 ExecStart=/opt/alist/alist (code=exited, status=1/FAILURE)
 Main PID: 25504 (code=exited, status=1/FAILURE)

Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] register driver: [WebDav]
Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] register driver: [Yandex.Disk]
Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] reading config file: data/config.json
Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] init cron...
Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] init model...
Feb 08 08:10:52 debian alist[25504]: INFO[2022-02-08 08:10:52] auto migrate model...
Feb 08 08:10:52 debian alist[25504]: ERRO[2022-02-08 08:10:52] open zhimg.html: file does not exist
Feb 08 08:10:52 debian alist[25504]: FATA[2022-02-08 08:10:52] open index.html: file does not exist
Feb 08 08:10:52 debian systemd[1]: alist.service: Main process exited, code=exited, status=1/FAILURE
Feb 08 08:10:52 debian systemd[1]: alist.service: Failed with result 'exit-code'.

**然后只好复制2.07二进制文件去覆盖后**
root@debian:~# systemctl start alist
root@debian:~# systemctl status alist
● alist.service - Alist service
   Loaded: loaded (/lib/systemd/system/alist.service; enabled; vendor preset: enabled)
   Active: active (running) since Tue 2022-02-08 08:11:31 EST; 1min 3s ago
 Main PID: 25515 (alist)
    Tasks: 7 (limit: 1099)
   Memory: 43.1M
   CGroup: /system.slice/alist.service
           └─25515 /opt/alist/alist

Feb 08 08:11:31 debian alist[25515]: INFO[2022-02-08 08:11:31] init settings...
Feb 08 08:11:32 debian alist[25515]: INFO[2022-02-08 08:11:32] init accounts...
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: aliyun, type: AliDrive
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: onedrive, type: Onedrive
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: sharepoint, type: Onedrive
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: qiniu, type: S3
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: storj, type: S3
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] success init account: NFS_tokyo, type: Native
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] init cache...
Feb 08 08:11:33 debian alist[25515]: INFO[2022-02-08 08:11:33] start server @ 0.0.0.0:7906

**Comment by xhofe** [comment]
善用搜索。

**Reply by iso-lib** [reply]
> 善用搜索。

哦耶,搜索解决了,升级到2.09要修改config.json中的配置assets为空值.

作者好勤快
